src/index.html

```diff
@@ -22,6 +22,508 @@
+
+          <p class="card__price-amount-style">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <button
+            class="card__buy-button"
+            type="button"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            class="card__image-photo"
+            src="images/imac.jpeg"
+            alt="product photo"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__product-code">
+          <p class="card__product-code-style">Product code: 195434</p>
+        </div>
+
+        <div class="card__grade">
+          <div class="card__grade-stars">
+            <ul class="card__grade-stars-list">
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+            </ul>
+          </div>
+
+          <div class="card__grade-reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-style">Price:</p>
+
+          <p class="card__price-amount-style">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <button
+            class="card__buy-button"
+            type="button"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image">
+          <img
+            class="card__image-photo"
+            src="images/imac.jpeg"
+            alt="product photo"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__product-code">
+          <p class="card__product-code-style">Product code: 195434</p>
+        </div>
+
+        <div class="card__grade">
+          <div class="card__grade-stars">
+            <ul class="card__grade-stars-list">
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+
+              <li class="card__grade-stars-item"></li>
+            </ul>
+          </div>
+
+          <div class="card__grade-reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-style">Price:</p>
+
+          <p class="card__price-amount-style">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <button
+            class="card__buy-button"
+            type="button"
+          >
+            BUY
+          </button>
+        </div>
+      </div>
```

Comment:
The content within these lines appears to be repeated multiple times with the same product details. Consider using a loop or a template in your backend or JavaScript to dynamically generate these cards if they are meant to be the same, or ensure each card has unique content if they are meant to be different.